### PR TITLE
Bump ui_test

### PR DESCRIPTION
**Pull request by oli-obk**
This should give a bunch of ui improvements when lots of tests are running. Please lmk of any issues with it. Switching to `--quiet` should always avoid any issues, so there's a workaround if anything crops up. Also please check that this does not regress performance for you. It doesn't for me, but that doesn't mean much.

changelog: none

### CODE REVIEW

File: tests/ui/mixed_attributes_style.rs [hunk 1]
```diff
@@ -82,7 +82,8 @@ mod issue_12530 {
             #![allow(dead_code)]
         }
     }
-    /// Nested mod //~ ERROR: item has both inner and outer attributes
+    /// Nested mod
+    //~^ ERROR: item has both inner and outer attributes
```
Comment (oli-obk) [inline tests/ui/mixed_attributes_style.rs:86]: you can't have annotations inside other comments anymore.

**Comment by alex-semenyuk** [comment]
Test failing 

```
FAILURES:
    tests/ui/almost_complete_range.fixed
```

**Comment by oli-obk** [comment]
Everything works now

**Comment by flip1995** [comment]
Alex is currently busy I think, so jumping in.

r? @flip1995

**Comment by flip1995** [comment]
Thanks!

@bors r+

**Pull request merged**